whaledo/transforms.py

```diff
@@ -175,7 +174,7 @@ def num_sources(self) -> int:
 @dataclass
 class MultiCropOutput(InputContainer[MultiViewPair]):
     global_views: MultiViewPair
-    local_views: Tensor
+    local_views: Optional[Tensor]
```

Comment:
Why Optional?

Reply:
Okay because of the __add__ method which is outdated now -- MultiViewPair is the form taken when local_views is None